apple/internal/partials/embedded_bundles.bzl

```diff
@@ -35,6 +43,10 @@ top-level bundling rule will need to package.""",
         "app_clips": """
 A depset with the zipped archives of bundles that need to be expanded into the
 AppClips section of the packaging bundle.""",
+        "custom_bundles": """
+A dict of depsets of zipped archives of bundles that need to be expanded into
+custom locations of the packaging bundle, where the key is the path to the
+custom location.""",
```

Comment:
This is needed to support the EarlGray use case where the bundle needs to be placed at a custom location within the packaging bundle.